src/timeline/TimelineViewManager.cpp

```diff
@@ -407,8 +407,8 @@ TimelineViewManager::verifyUser(QString userid)
         }
 
         emit ChatPage::instance()->showNotification(
-          tr("No share room with this user found. Create an "
-             "encrypted room with this user and try again."));
+          tr("No shared room with this user found. Create an "
+             "encrypted private chat or room with this user and try again."));
```

Comment:
I guess this should really be "Create an encrypted private chat with this user".